src/scss/components/_sidenav.scss

```diff
@@ -34,3 +34,14 @@
   padding-top: units(1.5);
   padding-bottom: units(1.5);
 }
+
+li > .active {
+  color: $theme-color-primary-dark;
+  font-weight: bold;
+}
+
+.sidenav {
+  ul {
+    border: none;
+  }
+}
```

Comment:
In the `.usa-sidenav` styles above, we add a border style. If the expectation is that the LGDS sidenav has no border, could we remove it there?

Reply:
Yes. I will remove that border.